R/environment.r

```diff
@@ -44,3 +44,20 @@ init_shadowenv <- function() {
         stop("'edit()' not yet supported in the Jupyter R kernel")
     })
 }
+
+init_cran_repo <- function() {
+    r <- getOption('repos')
+    is_configured_by_user <- !identical(r, c(CRAN = '@CRAN@'))
+    if (!is_configured_by_user) {
```

Comment:
I don't agree: at least to me it's more meaningful to name it `is_configured_by_user` instead of  ìs_default. Maybe it could be "is_useable_mirror', but not `is_default` (which I would bet on is changed in one of the next versions to that mirror as well :-))